### DISCUSSION TITLE
Enforcing a trailing-slash policy

### DISCUSSION THREAD

**Discussion by ryczek02** [tighten/ziggy#496] [Q&A] [answered]
#### Description

Is there a way to force a trailing slash policy in ziggy? My .htaccess does 301 to a link with '/' at the end, however I would like the links on the page to appear with '/' at the end. Unfortunately 

`route('') + '/'`

does not always work well.

I would like to do this in the app.js file (my main javascript file)

Thanks in advance for your reply. Greetings 

![image](https://user-images.githubusercontent.com/49419559/148541729-fcbd23b5-8db2-4143-a79e-b591f2314b95.png)

**Comment by bakerkretzmar** [comment, marked as answer]
There isn't a way to do this within Ziggy itself, no, but you can just wrap Ziggy's `route()` function to do it yourself.

Something like this should work, but let me know if you still have questions!

```js
// Assuming the `route()` function already exists...

appRoute = (...args) => route(...args) + '/';
```

**Reply by ryczek02** [reply]
Thank you so much. 

If anyone wants to know for Inertia.js my part of the script looks like this:

```js
createInertiaApp({
    title: (title) => `${title}`,
    resolve: (name) => require(`./Pages/${name}.vue`),
    setup({ el, app, props, plugin }) {
        return createApp({ render: () => h(app, props) })
            .use(plugin)
            .mixin({ methods: {
                route,
                addRoute: (...args) => route(...args) + '/', // bakerkretzmar Reply 
```

**Reply by bakerkretzmar** [reply]
Oh cool okay if you're setting up the mixin manually like that you can call it just `route`:

```js
    // ...
    .mixin({
        methods: {
            route: (...args) => route(...args) + '/',
        },
    })
```
